sce.bat

```diff
@@ -144,6 +144,19 @@ REM set the varible %name% to the resolved repo.
         goto :print_repo_not_found
     )
     cd %REPO_LOCATION%
+    REM Check if config file exists before running
+    IF %name% == %CLARK_REPO_NAME% (
+        IF NOT EXIST "./src/config/config.json" (
+            set "location=%REPO_LOCATION%\src\config\"
+            goto :print_missing_config
+        )
+    )
+    IF %name% == %SCE_DISCORD_BOT_REPO_NAME% (
+        IF NOT EXIST "./config.json" (
+            set "location=%REPO_LOCATION%\"
+            goto :print_missing_config
+        )
+    )
```

Comment:
i think this is cleaner, lmk if it makes sense/works
```suggestion
    REM Default to the config existing at the root of the project.
    REM The path is relative because at this point, we are already
    REM in the directory of the specified project.
    SET CONFIG_LOCATION="./config.json"
    IF %name% == %CLARK_REPO_NAME% (
        SET CONFIG_LOCATION="./src/config/config.json"
    )
    IF NOT EXIST %CONFIG_LOCATION% (
        set "location=%REPO_LOCATION%\"
        goto :print_missing_config
    )
```